tests/admin_views/admin.py

```diff
@@ -881,8 +881,16 @@ class CityInlineAdmin(admin.TabularInline):
     view_on_site = False
 
 
+class StateAdminForm(forms.ModelForm):
```

Comment:
If there's a purpose for `label_form_field` at this point, I can't easily spot it.

Reply:
Thanks for pointing this out Tim. I actually forgot to check this in the test and as soon as I added this, I discovered another issue.

We were doing: 
`verbose_field_name = (form.fields[field_name].label) or field_name`

which would make `field_name` unreachable. Since, even if `form.fields[field_name].label` is `None`, the stringify version of it would always evaluate to `True`.
I'm fixing this as well as the similar change in case formsets are there.